roles/ncn-common/files/scripts/common/create-ims-initrd.sh

```diff
@@ -31,6 +31,7 @@ set -ex
 echo "Generating initrd..."
 dracut \
 --force \
+--force-add dmsquash-live \
```

Comment:
Change this to `"dmsquash-live livenet"`, this'll help us support rebuilds of the UAN since they also use what we vendor into node-images.

`dmsquash-live` brings in support for `root=live:[disk]`, and `livenet` brings support for `root=live:[http-url]`.